authorization-common/README.md

```diff
@@ -118,11 +114,12 @@ Some APIs are used internal only between services, in this case we are using int
 
 #### But what about account id?
 
-We are using account id in service layer heavily, but internal actor CRN doesn't have that information, so there are specific requirements for APIs which can called with internal actor:
+We are using account id in service layer heavily, but internal actor CRN doesn't have that information, so there are specific requirements for APIs which can be called with internal actor:
 
 Your method in the Controller class should have at least one parameter:
 - which is some kind of resource CRN -> annotate the parameter with `@TenantAwareParam`, which will extract the account id from CRN parameter and use that in lower layers
-- or it is an accound id parameter -> annotate a CRN parameter with `@AccountId` and we'll use that in lower layers. Please note that `@AccountId` is also a validation annotation, thus you need to place it on the API interface method's parameter.
+- or it is an accound id parameter -> annotate the parameter with `@AccountId` and we'll use that in lower layers. Please note that `@AccountId` is also a validation annotation, thus you need to place it on the API interface method's parameter.
```

Comment:
Please fix this typo `accound id` to `account id`

Reply:
thx, fixed